pandas/core/internals/blocks.py

```diff
@@ -1363,6 +1363,7 @@ def where(
         errors : str, {'raise', 'ignore'}, default 'raise'
             - ``raise`` : allow exceptions to be raised
             - ``ignore`` : suppress exceptions. On error return original object
+        try_cast: bool, default False
         axis : int, default 0
```

Comment:
What this parameter ``try_cast`` is for? Looks like it is not used in the function. It will be better for the code cleanup if you delete it from the function signature and everywhere it gets called.

Reply:
Good catch

Reply:
Sure, I saw quite some of these arguments in other functions as well (ones which are not used), but when I remove them, some tests fail. So would it be safe to remove those in the tests or the tests themselves as well?

Reply:
It is hard to say why tests fail in this case without looking at the traceback.
Need to check whether the test itself calls the method with all the arguments, or the method is called somewhere else. In the latter case one needs to change all calls of the method (that could probably be called in multiple places).

Reply:
if its just that the tests pass the unused arg directly, then its fine to remove the arg from the test too.

If you've got some easy things (like this PR so far) and some less-obvious things, usually best to keep them as separate PRs.